data/data.json

```diff
@@ -3,6 +3,14 @@
       "text":"The only people who never fail are those who never try.",
       "from":"Ilka Chase"
    },
+   {
+    "quote": "A ship in port is safe, but that's not what ships are built for.",
```

Comment:
it's `text`, not `quote`

Reply:
also, it appears as though John A. Shedd should be credited, not Grace Hopper.

https://quoteinvestigator.com/2013/12/09/safe-harbor/

>In conclusion, QI believes that John A. Shedd should be credited with the quotation he employed in the 1928 volume “Salt from My Attic”. Grace M. Hopper did use the expression, and she helped to popularize it, but she did not coin it. The connection to Albert Einstein appeared late and was not substantive.